### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by tpiros** [denosaurs/denon#125]
**Describe the bug**
Using Deno 1.7.1 and I can't install denon. See below

**To Reproduce**

`deno install -qAf --unstable https://deno.land/x/denon/denon.ts`
```
error: TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname
                 ~~~
    at https://deno.land/std@0.53.0/path/win32.ts:911:18

TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname;
                 ~~~
    at https://deno.land/std@0.53.0/path/posix.ts:433:18

TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
export { ParseOptions, parse, parseAll } from "./_yaml/parse.ts";
         ~~~~~~~~~~~~
    at https://deno.land/std@0.53.0/encoding/yaml.ts:6:10

TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
  DumpOptions as StringifyOptions,
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/std@0.53.0/encoding/yaml.ts:8:3

TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
export { SchemaDefinition } from "./_yaml/schema.ts";
         ~~~~~~~~~~~~~~~~
    at https://deno.land/std@0.53.0/encoding/yaml.ts:11:10

TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
export { StyleVariant } from "./_yaml/type.ts";
         ~~~~~~~~~~~~
    at https://deno.land/std@0.53.0/encoding/yaml.ts:12:10

TS2339 [ERROR]: Property 'url' does not exist on type 'NetPermissionDescriptor'.
              return pd.url
                        ~~~
    at https://deno.land/std@0.53.0/permissions/mod.ts:16:25

TS2339 [ERROR]: Property 'url' does not exist on type 'NetPermissionDescriptor'.
                ? `--allow-${pd.name}=${pd.url}`
                                           ~~~
    at https://deno.land/std@0.53.0/permissions/mod.ts:17:44

TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
  LevelName as LogLevelName,
  ~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/denon/deps.ts:8:3

TS2790 [ERROR]: The operand of a 'delete' operator must be optional.
    delete g.scripts;
           ~~~~~~~~~
    at https://deno.land/x/denon/src/runner.ts:48:12

Found 10 errors.
```
**Expected behavior**
The installation to work.

**Setup**

- OS: Latest Mac OS
- Deno version: (`deno version`) 1.7.1
- Denon version: (`denon --version`) n/a

**Comment by freshfish70** [comment]
I'm also running the latest Mac OS, but have been using deno 1.6.3  and denon 2.4.6. 
A patch( #124) was released to be compatible with deno 1.7.x. 

Just upgraded to deno 1.7.1 and installed denon 2.4.7 without issues. Did you just upgrade?
- Try manually delete old denon install before upgrading, event though -f should overwrite it.
- Set version tag when pulling ```deno install--unstable https://deno.land/x/denon@2.4.7/denon.ts```

 
![image](https://user-images.githubusercontent.com/19345328/106917713-d1ab5d00-6708-11eb-9986-26e65d24ab77.png)

**Comment by eliassjogreen** [comment]
You are by the looks of it using an outdated version of denon, upgrade by running
```
deno install -qAf --reload --unstable https://deno.land/x/denon@2.4.7/denon.ts
```
when using deno 1.7.1
